mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/bridge/binding/IdentifierBridgeBindingContext.java

```diff
@@ -16,6 +16,8 @@
 
 	/**
 	 * @return An entry point allowing to inspect the type of values that will be passed to this bridge.
+	 * @hsearch.experimental This feature is under active development.
+	 *    You should be prepared for incompatible changes in future releases.
```

Comment:
Maybe using a 'you' in a Javadoc could be too much informal.
What do you think of `The API is is susceptible to change` ?

Reply:
That's a message dating back to Search 5, but ok.
I changed it to: `Usual compatibility policies do not apply: incompatible changes may be introduced in any future release.`